js/src/MenuButton.js

```diff
@@ -0,0 +1,50 @@
+/** @jsx h */
+import { h, Component } from 'preact'
+import '../../css/src/dropdown.css'
+
+const _ = require('underscore')
+
+class MenuButton extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      disabled: _.contains(props.disabledButtons, props.name)
```

Comment:
I'd rather use `Array.prototye.includes` instead of underscore: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
_.contains will return false if the target is null. Array.prototype.includes will throw an error.